src/rocdecode/rocdecode_api.cpp

```diff
@@ -193,6 +193,30 @@ rocDecGetVideoFrame(rocDecDecoderHandle decoder_handle, int pic_idx,
     return ret;
 }
 
+/************************************************************************************************************************/
+//! \fn extern rocDecStatus ROCDECAPI rocDecReleaseVideoFrame(rocDecDecoderHandle decoder_handle, int pic_idx)
+//! \ingroup group_amd_rocdecode
+//! Release video frame corresponding to pic_idx for reuse after consuming the interopped device memory
+/************************************************************************************************************************/
+extern rocDecStatus ROCDECAPI rocDecReleaseVideoFrame(rocDecDecoderHandle decoder_handle, RocdecVideoParser parser_handle, int pic_idx) {
```

Comment:
As explained above, this new API at the decoder level is not needed. Please remove it.